tests/commands/ironhawk/incr_test.go

```diff
@@ -6,165 +6,111 @@ package ironhawk
 import (
```

Comment:
In this file, I see a lot of conversions happening `int64(*)`. See if we can get rid of it. Most should work without an explicit typecast.